cirq-core/cirq/devices/thermal_noise_model.py

```diff
@@ -283,3 +313,31 @@ def noisy_moment(self, moment: cirq.Moment, system_qubits: Sequence[cirq.Qid]) -
             return [moment]
         output = [moment, moment_module.Moment(noise_ops)]
         return output[::-1] if self._prepend else output
+
+    def _json_dict_(self) -> dict[str, object]:
+        gate_times = {cirq.json_cirq_type(k): v for k, v in self.gate_durations_ns.items()}
+        return {
+            'gate_durations_ns': tuple(gate_times.items()),
+            'rate_matrix_GHz': tuple((q, m.tolist()) for q, m in self.rate_matrix_GHz.items()),
```

Comment:
`rate_matrix_GHz` has derived values which should not be stored.  Please use it only to recover the `qubits` for class init, which do not seem to be stored anywhere else.  Otherwise, just store here only the init arguments with a minimum conversions needed.

Reply:
Fixed in 0a1332d780a300be4395f2120242f4ffc36f1d0e